packages/builder/src/pages/builder/portal/manage/auth/index.svelte

```diff
@@ -9,58 +15,159 @@
     Layout,
     Input,
     Body,
+    Select,
   } from "@budibase/bbui"
   import { onMount } from "svelte"
   import api from "builderStore/api"
+  import { writable } from "svelte/store"
+  import { organisation } from "stores/portal"
+
+  const values = writable({
+    oidcIcon: $organisation.oidcIcon,
+    oidcName: $organisation.oidcName,
+  })
 
   const ConfigTypes = {
     Google: "google",
+    OIDC: "oidc",
     // Github: "github",
     // AzureAD: "ad",
   }
 
-  const ConfigFields = {
+  const GoogleConfigFields = {
     Google: ["clientID", "clientSecret", "callbackURL"],
   }
-  const ConfigLabels = {
+  const GoogleConfigLabels = {
     Google: {
       clientID: "Client ID",
       clientSecret: "Client secret",
       callbackURL: "Callback URL",
     },
   }
 
+  const OIDCConfigFields = {
+    Oidc: ["configUrl", "clientID", "clientSecret"],
+  }
+  const OIDCConfigLabels = {
+    Oidc: {
+      configUrl: "Config URL",
+      clientID: "Client ID",
+      clientSecret: "Client Secret",
+    },
+  }
+
+  let iconDropdownOptions = [
+    {
+      label: "Azure AD",
+      value: "AD",
+      icon: MicrosoftLogo,
+    },
+    { label: "Oracle", value: "Oracle", icon: OracleLogo },
```

Comment:
The oracle logo doesn't look that great when used on the login, particularly on dark mode. I'd be up for removing it to improve the look of the product